classes/class-wc-connect-shipping-method.php

```diff
@@ -363,7 +363,7 @@ public function calculate_shipping( $package = array() ) {
 						'label'     => self::format_rate_title( $rate->title ),
 						'cost'      => $rate->rate,
 						'meta_data' => array(
-							'wc_connect_packages' => json_encode( $rate->packages ),
+							'wc_connect_packages' => $rate->packages,
```

Comment:
You're changing this metadata from a string to an array / object. I'm thinking of users when they upgrade, will they end up with differently encoded `wc_connect_packages` in their meta data tables?